app.js

```diff
@@ -1,18 +1,30 @@
 const express = require('express');
 const { getTopics } = require('./controller/topics.contoller');
-const { getArticleById } = require('./controller/articles.controller');
+const { getArticleById, patchArticleById } = require('./controller/articles.controller');
 const app = express();
 
+app.use(express.json())
+
 app.get('/api/topics', getTopics);
 
 app.get('/api/articles/:article_id', getArticleById)
 
+app.patch('/api/articles/:article_id', patchArticleById)
+
 app.use((err, req, res, next) => {
    if(err.code === '22P02'){
        res.status(400).send({message: 'Bad request'})
    } else { 
        next(err) }
 })
+
```

Comment:
Try to remember DRY, these two error handlers do a ver similar job, can you think of a refactor that would make it less repetitive?